Add minimum profile README badge. (#4)
miyo-reader/.github: profile/README.md

## Hi there 👋
This is the :octocat: home of the [Miyo Reader](https://www.miyo-reader.io) project.

> It's purpose is to build a **open** software/open hardware **ebook reader**.

Reviewer: ```suggestion
Its purpose is to build an **open** software/open hardware **ebook reader**.
```
